lib/signaling/v2/peerconnection.js

```diff
@@ -1159,4 +1183,46 @@ function shouldStopTransceiver(sdp, transceiver) {
   return false;
 }
 
+/**
+ * Perform certain updates after an SDP negotiation is completed.
+ * @param {PeerConnectionV2} pcv2
+ * @returns {void}
+ */
+function negotiationCompleted(pcv2) {
```

Comment:
we should make this private function  of the  class `PeerConnectionV2` - Looks like we only callers are passing `this` to this function

Reply:
@makarandp0 Let's try to minimize the number of "private" methods on a particular class.

Reply:
I think we should follow simple rules to decide when to make a function `private` v/s file scoped. In this case given that function does not access outside `this`, it makes sense for it to be private. I see your concern about this class growing, I guess we need to find better way to break it down.